wagtailinventory/views.py

```diff
@@ -1,35 +1,65 @@
-from django.core.paginator import Paginator
-from django.http import HttpResponseBadRequest
-from django.shortcuts import render
-from django.views.generic import View
+from django import forms
 
+from wagtail.admin.filters import WagtailFilterSet
+from wagtail.admin.views.reports import PageReportView
 from wagtail.models import Page
 
-from wagtailinventory.forms import PageBlockQueryFormSet
+import django_filters
 
+from wagtailinventory.checks import dal_select2_check_all
+from wagtailinventory.models import PageBlock
 
-class SearchView(View):
-    template_name = "wagtailinventory/search.html"
 
-    def get(self, request):
-        if len(request.GET) > 1:
-            formset = PageBlockQueryFormSet(request.GET)
-            if not formset.is_valid():
-                return HttpResponseBadRequest("invalid query")
+# If django-autocomplete-light is present, we'll do two things:
+#
+# 1. Set the widget used for the block include/exclude filters to DAL's
+#    multi-select;
+# 2. Create a list view populated with available blocks to autocomplete.
+#
+# If it's not present, we'll fall back on Django's built-in SelectMultiple
+# widget.
+if not dal_select2_check_all():  # pragma: no cover
+    autocomplete = None
+    block_widget = forms.SelectMultiple()
+else:
+    from dal import autocomplete
 
-            queryset = formset.get_query()
-        else:
-            formset = PageBlockQueryFormSet()
-            queryset = Page.objects.all()
+    block_widget = autocomplete.Select2Multiple(url="block-autocomplete")
 
-        queryset = queryset.order_by("title")
```

Comment:
Was this an intentional change? The existing wagtail-inventory logic sorts results by title; I can see an argument that's arbitrary but not sure if you meant to remove that here.

Reply:
Ah, I think I didn't port this over! I'm not certain an `order_by` on the queryset itself is the best way in a report, but I'll ensure we're sorting on title.

Reply:
Fixed in [267c3fc](https://github.com/cfpb/wagtail-inventory/pull/68/commits/267c3fcce41b2f6f5eaa557aa8093f47158e2653).